admin/src/Accounts/AccountList.tsx

```diff
@@ -1,12 +1,20 @@
-import { Datagrid, List, TextField } from "react-admin";
+import { DataTable, List, ReferenceManyCount } from "react-admin";
 
 export function AccountList() {
 	return (
 		<List resource="accounts">
-			<Datagrid>
-				<TextField source="name" />
-				<TextField source="email" />
-			</Datagrid>
+			<DataTable hiddenColumns={["id"]}>
```

Comment:
Much better but there's no way to show the `id` column: https://marmelab.com/react-admin/DataTable.html#hiding-or-reordering-columns